rest/endpoints/ceph.py

```diff
@@ -0,0 +1,66 @@
+import json
+
+from rest.common.config.config import Config
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class CEPH:
+    def __init__(self, rest):
+        """
+        Constructor for CEPH related REST API calls
+        """
+        self._config = Config()
+        self._rest = rest
+
+    def create_pool(self, **kw):
```

Comment:
Does it work for both Replicated and EC pool?

Reply:
its an REST endpoint for creating any pool, depends on data one can create EC pool too. I have added request details in doc string for reference